SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -1,7 +1,11 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 #pragma once
 
+#include "Containers/Queue.h"
+#include "HAL/Runnable.h"
+#include "HAL/ThreadSafeBool.h"
```

Comment:
Separate engine includes from GDK ones.
```suggestion
#include "HAL/ThreadSafeBool.h"

```